### PR TITLE
Stm32mp1 new bsp support (minimal intrusive changes)

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

File: meta-lmp-base/conf/layer.conf [hunk 1]
```diff
@@ -20,6 +20,7 @@ LAYERDEPENDS_meta-lmp-base = " \
     security \
 "
 LAYERSERIES_COMPAT_meta-lmp-base = "kirkstone"
+LAYERSERIES_COMPAT_stm-st-stm32mp:append = " kirkstone"
```
Comment (quaresmajose) [inline meta-lmp-base/conf/layer.conf:23]: meta-lmp-bsp/conf/layer.conf is more apropriate for this imo

Comment (igoropaniuk) [reply meta-lmp-base/conf/layer.conf]: fixed

**Review by angolini — APPROVED** [review]
LGTM

**Review by quaresmajose — APPROVED** [review]

**Review by MrCry0 — APPROVED** [review]

File: meta-lmp-base/conf/distro/include/lmp.inc [hunk 2]
```diff
@@ -119,6 +119,8 @@ BBMASK += " \
     /meta-st-stm32mp/recipes-graphics/drm/libdrm_2.4.110.bbappend \
     /meta-st-stm32mp/recipes-graphics/drm/libdrm_2.4.113.bbappend \
     /meta-st-stm32mp/recipes-bsp/trusted-firmware-a/tf-a-stm32mp_2.6.bb \
+    /meta-st-stm32mp/recipes-graphics/drm/libdrm_2.4.115.bbappend \
+    /meta-st-stm32mp/recipes-devtools/gcc/gcc-source_12.2.bbappend \
```
Comment (ricardosalveti) [inline meta-lmp-base/conf/distro/include/lmp.inc:123]: We can probably drop the first 2 lines (libdrm) and replace with the newer version instead.

**Comment by ricardosalveti** [comment]
What is the meta-st revision you are using together with these changes?

**Comment by igoropaniuk** [comment]
@ricardosalveti https://github.com/STMicroelectronics/meta-st-stm32mp/commit/931b3f5be2c00f55d114c8f9122082a9d229b776

File: meta-lmp-base/conf/distro/include/lmp.inc (hunk 2 above)
Comment (igoropaniuk) [reply meta-lmp-base/conf/distro/include/lmp.inc:119]: done

**Comment by ricardosalveti** [comment]
Changes are ok, will validate with the proposed hash for meta-st tomorrow.

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**